core/types/block.go

```diff
@@ -85,29 +86,36 @@ func (n *BlockNonce) UnmarshalText(input []byte) error {
-	utxoRoot      common.Hash     `json:"utxoRoot"               gencodec:"required"`
-	txHash        common.Hash     `json:"transactionsRoot"     gencodec:"required"`
-	etxHash       common.Hash     `json:"extTransactionsRoot"  gencodec:"required"`
-	etxSetHash    common.Hash     `json:"etxSetHash"        gencodec:"required"`
-	etxRollupHash common.Hash     `json:"extRollupRoot"        gencodec:"required"`
-	manifestHash  []common.Hash   `json:"manifestHash"         gencodec:"required"`
-	receiptHash   common.Hash     `json:"receiptsRoot"         gencodec:"required"`
-	difficulty    *big.Int        `json:"difficulty"           gencodec:"required"`
-	parentEntropy []*big.Int      `json:"parentEntropy"        gencodec:"required"`
-	parentDeltaS  []*big.Int      `json:"parentDeltaS"         gencodec:"required"`
-	number        []*big.Int      `json:"number"               gencodec:"required"`
-	gasLimit      uint64          `json:"gasLimit"             gencodec:"required"`
-	gasUsed       uint64          `json:"gasUsed"              gencodec:"required"`
-	baseFee       *big.Int        `json:"baseFeePerGas"        gencodec:"required"`
-	location      common.Location `json:"location"             gencodec:"required"`
-	time          uint64          `json:"timestamp"            gencodec:"required"`
-	extra         []byte          `json:"extraData"            gencodec:"required"`
-	mixHash       common.Hash     `json:"mixHash"              gencodec:"required"`
-	nonce         BlockNonce      `json:"nonce"`
+	parentHash            []common.Hash   `json:"parentHash"            gencodec:"required"`
+	uncleHash             common.Hash     `json:"sha3Uncles"            gencodec:"required"`
+	coinbase              common.Address  `json:"miner"                 gencodec:"required"`
+	evmRoot               common.Hash     `json:"evmRoot"               gencodec:"required"`
+	utxoRoot              common.Hash     `json:"utxoRoot"              gencodec:"required"`
+	txHash                common.Hash     `json:"transactionsRoot"      gencodec:"required"`
+	etxHash               common.Hash     `json:"extTransactionsRoot"   gencodec:"required"`
+	etxSetHash            common.Hash     `json:"etxSetHash"            gencodec:"required"`
+	etxRollupHash         common.Hash     `json:"extRollupRoot"         gencodec:"required"`
+	manifestHash          []common.Hash   `json:"manifestHash"          gencodec:"required"`
+	receiptHash           common.Hash     `json:"receiptsRoot"          gencodec:"required"`
+	difficulty            *big.Int        `json:"difficulty"            gencodec:"required"`
+	parentEntropy         []*big.Int      `json:"parentEntropy"         gencodec:"required"`
+	parentDeltaS          []*big.Int      `json:"parentDeltaS"          gencodec:"required"`
+	parentUncledSubDeltaS []*big.Int      `json:"parentUncledSubDeltaS" gencodec:"required"`
+	efficiencyScore       uint16          `json:"efficiencyScore"       gencodec:"required"`
```

Comment:
I think you've added too many fields now. For example, `efficiencyScore` & `etxEligibleSlices` are deterministic. Any honest node following the protocol will have these values without needing to read them from the header.

I could say the same about the parentEntropy/Delta/UncledDelta/etc values

Reply:
I think you are okay with leaving this in header for now as when the work object comes in, it will only have a commitment to all this information

Reply:
sure, we can leave it for now since PoEM already blew this out of the water. We can revisit later to remove all this unnecessary data from the header.